entity/dag/entity_actions_test.go

```diff
@@ -406,3 +406,36 @@ func TestRemove(t *testing.T) {
 	err = Remove(def, repoA, e.Id())
 	require.NoError(t, err)
 }
+
+func TestRemoveAll(t *testing.T) {
+	type args struct {
+		def  Definition
+		repo repository.ClockedRepo
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "foo",
+			args: args{
+				def: Definition{
+					Typename:             "foo",
+					Namespace:            "foo",
+					OperationUnmarshaler: nil,
+					FormatVersion:        0,
+				},
+				repo: repository.NewMockRepo(),
+			},
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if err := RemoveAll(tt.args.def, tt.args.repo); (err != nil) != tt.wantErr {
```

Comment:
Unless I'm missing something, you are executing `RemoveAll` on an empty repo. Arguably that's not testing very much.

Reply:
Take example on `TestRemove` in the same file to generate some data ?

Reply:
Yeah, you're right. I added data at the beginning of the test.